docs/user_guide/vectors_and_matrices.mdx

```diff
@@ -501,3 +501,69 @@ compile-time.
 ![Fixed resizing](/img/resizing.svg)
 
 </div>
+
+## Matrix Macros 
+### Macros 
```

Comment:
I think maybe we should remove hte `### Macros` header because it just causes unnecessary nesting in the navigation (on the right side).

Reply:
Ok, removed the header.